change-beta/@azure-communication-react-6d0bac59-3894-4586-aaa0-ee9b817fcee7.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "none",
```

Comment:
Should this be updated too? (as we don't support `none` type anymore)

Reply:
Updated